app/_components/SubmissionModal.tsx

```diff
@@ -290,7 +290,10 @@ export const SubmissionModal = ({
 					}`
 				);
 			} else {
-				setError(e?.cause?.message || e?.message || 'An error occurred');
+				if (e.message.includes('User rejected the request')) {
+					return setError('User rejected the transaction.');
+				}
+				setError('An error occurred');
```

Comment:
this isn't a useful error though, we still need the error rmessage to be available somehow

Reply:
Do we put the full error message in a dropdown?

Reply:
Using error.details instead of a custom error returning eg:

<img width="633" alt="image" src="https://github.com/user-attachments/assets/a3c87076-a667-43b2-8dd1-6019b05a415a">

Reply:
ok, let's include the details here too